test/views/StartViewSpec.scala

```diff
@@ -18,138 +18,133 @@ package views
 
 import helpers.views.declaration.StartMessages
 import play.twirl.api.Html
-import views.declaration.spec.AppViewSpec
+import unit.tools.Stubs
+import views.declaration.spec.UnitViewSpec
 import views.html.start_page
 import views.tags.ViewTest
 
 @ViewTest
-class StartViewSpec extends AppViewSpec with StartMessages {
+class StartViewSpec extends UnitViewSpec with StartMessages with Stubs {
 
-  private val startPage = app.injector.instanceOf[start_page]
+  private val startPage = new start_page(mainTemplate)
   private def createView(): Html = startPage()
 
   "Start View on empty page" when {
 
     "display page title" in {
 
-      createView().select("title").text() must be(messages(title))
+      createView().select("title").text() mustBe messages(title)
```

Comment:
I don't understand what you mean here

Reply:
It is little redundant to call stubbed messages as same - maybe having testing messages which would preserve key but slightly modify it would be good ?

Reply:
At the moment we have tests that test the content of a message and others that test that the message key is present.
We can continue with both approaches for the time being.